2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_setup.py:_flush():76] Configure stats pid to 3221425
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_setup.py:_flush():76] Loading settings from /home/anwer/.config/wandb/settings
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_setup.py:_flush():76] Loading settings from /12tb_dsk3/SEIR_model/src/wandb/settings
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'sweeps-SEIR', 'entity': 'danish2562022', 'root_dir': '/12tb_dsk3/SEIR_model/src', 'run_id': '4sa8870s', 'sweep_param_path': '/12tb_dsk3/SEIR_model/src/wandb/sweep-2gowftbl/config-4sa8870s.yaml', 'sweep_id': '2gowftbl'}
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:_log_setup():529] Logging user logs to /12tb_dsk3/SEIR_model/src/wandb/run-20250526_180234-4sa8870s/logs/debug.log
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:_log_setup():530] Logging internal logs to /12tb_dsk3/SEIR_model/src/wandb/run-20250526_180234-4sa8870s/logs/debug-internal.log
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:_jupyter_setup():475] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x77c56c9c5010>
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():569] calling init triggers
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():576] wandb.init called with sweep_config: {'ah0': 2.9989022536125645, 'ah1': 2.722618074090376, 'bh0': 1.9028408019141376, 'bh1': 1.4295482501402024, 'c': 1.277272459331144}
config: {}
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():619] starting backend
2025-05-26 18:02:34,256 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():623] setting up manager
2025-05-26 18:02:34,257 INFO    Thread-41 (_run_job):3221425 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-26 18:02:34,258 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():631] backend started and connected
2025-05-26 18:02:34,262 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_config_callback():1382] config_cb None None {'ah0': 2.9989022536125645, 'ah1': 2.722618074090376, 'bh0': 1.9028408019141376, 'bh1': 1.4295482501402024, 'c': 1.277272459331144}
2025-05-26 18:02:34,262 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_label_probe_notebook():1334] probe notebook
2025-05-26 18:02:34,262 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_label_probe_notebook():1344] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-05-26 18:02:34,262 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():720] updated telemetry
2025-05-26 18:02:34,265 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2025-05-26 18:02:34,640 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_on_init():2402] communicating current version
2025-05-26 18:02:34,717 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_on_init():2411] got version response upgrade_message: "wandb version 0.19.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-05-26 18:02:34,717 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():804] starting run threads in backend
2025-05-26 18:02:35,435 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_console_start():2380] atexit reg
2025-05-26 18:02:35,435 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2025-05-26 18:02:35,435 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_redirect():2300] Wrapping output streams.
2025-05-26 18:02:35,435 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_redirect():2325] Redirects installed.
2025-05-26 18:02:35,435 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:init():847] run started, returning control to user process
2025-05-26 18:02:35,516 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_finish():2109] finishing run danish2562022/sweeps-SEIR/4sa8870s
2025-05-26 18:02:35,516 INFO    Thread-41 (_run_job):3221425 [jupyter.py:save_history():444] not saving jupyter history
2025-05-26 18:02:35,516 INFO    Thread-41 (_run_job):3221425 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-05-26 18:02:35,516 INFO    Thread-41 (_run_job):3221425 [wandb_init.py:_jupyter_teardown():457] cleaning up jupyter logic
2025-05-26 18:02:35,516 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_atexit_cleanup():2349] got exitcode: 0
2025-05-26 18:02:35,516 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_restore():2332] restore
2025-05-26 18:02:35,516 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_restore():2338] restore done
2025-05-26 18:02:38,682 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_footer_history_summary_info():3991] rendering history
2025-05-26 18:02:38,683 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_footer_history_summary_info():4023] rendering summary
2025-05-26 18:02:38,685 INFO    Thread-41 (_run_job):3221425 [wandb_run.py:_footer_sync_info():3950] logging synced files
